create-app.ts

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import retry from 'async-retry'
+import chalk from 'chalk'
+import cpy from 'cpy'
+import fs from 'fs'
+import os from 'os'
+import path from 'path'
+import {
+  downloadAndExtractExample,
+  downloadAndExtractRepo,
+  getRepoInfo,
+  hasExample,
+  hasRepo,
+  RepoInfo,
+} from './helpers/examples'
+import { makeDir } from './helpers/make-dir'
+import { tryGitInit } from './helpers/git'
+import { install } from './helpers/install'
+import { isFolderEmpty } from './helpers/is-folder-empty'
+import { getOnline } from './helpers/is-online'
+import { shouldUseYarn } from './helpers/should-use-yarn'
+import { isWriteable } from './helpers/is-writeable'
+
+export class DownloadError extends Error {}
+
+export async function createApp({
+  appPath,
+  useNpm,
+  example,
+  examplePath,
+  typescript,
+}: {
+  appPath: string
+  useNpm?: boolean
+  example?: string
+  examplePath?: string
+  typescript?: boolean
+}): Promise<void> {
+  let repoInfo: RepoInfo | undefined
+  const template = typescript ? 'typescript' : 'default'
+  example = 'https://github.com/henripal/create-dao/tree/main/template'
```

Comment:
We'll be merging this in so let's update this to:

```suggestion
  example = 'https://github.com/Developer-DAO/create-dao/tree/main/template'
```